benchmarks/LIA/2013.OOPSLA_Hola/find.sl

```diff
@@ -0,0 +1,33 @@
+#need to change this based on having a aux var that tracks if we found
```

Comment:
Is this done?

Reply:
Not done, i will remove this from this directory and fix it.